server_api.py

```diff
@@ -21,6 +21,8 @@
 from apiclient import discovery
 import datetime
 from dataAccessObjects import UserDao
+from dataAccessObjects import ScheduleDao
```

Comment:
Please import the class in a line, such as 
`from dataAccessObjects import UserDao,ScheduleDao,ImageDao`